Scripts/datahandling/zonedata.py

```diff
@@ -76,6 +78,16 @@ def _add_transformations(self, data):
             if col.startswith("sh_wrk_"):
                 self[col.replace("sh_wrk_", "")] = data[col] * wp
         self.share["share_age_7-99"] = share_7_99
+        # Convert household shares to population shares
+        hh_population = (self["sh_hh1"] + 2*self["sh_hh2"] + 4.13*self["sh_hh3"])
+        self["sh_hh1"] = divide(self["sh_hh1"], hh_population)
+        self["sh_hh2"] = divide(2*self["sh_hh2"], hh_population)
+        self["sh_hh3"] = divide(4.13*self["sh_hh3"], hh_population)
+        self["sh_cars1_hh1"] = divide(self["sh_cars1_hh1"], hh_population)
+        self["sh_cars1_hh2"] = divide(2*self["sh_cars1_hh2"] + 4.13*self["sh_cars1_hh3"], 
+                                      hh_population)
+        self["sh_cars2_hh2"] = divide(2*self["sh_cars2_hh2"] + 4.13*self["sh_cars2_hh3"], 
+                                      hh_population)
```

Comment:
I am a little bit skeptical about having this calculation in the model system instead of zone-data creation process.